database/migrations/20190221120620_foreignkey.js

```diff
@@ -0,0 +1,10 @@
+
+exports.up = function(knex, Promise) {
+ return knex.schema.table('contacts', (table) => {
+  table.integer('contacts_id').references('id').inTable('users');
+ }) 
+};
+
+exports.down = function(knex, Promise) {
+  return knex.schema.dropColumn('contacts_id')
```

Comment:
You shouldn't be blindly dropping this table. You should be removing the reference in the `contacts_id` column.